pyenv-win/bin/pyenv.bat

```diff
@@ -1,16 +1,93 @@
 @echo off
-setlocal enableDelayedExpansion
+setlocal
 chcp 1250 >nul
 
-IF EXIST "%~dp0"..\exec.bat (
-   del /F /Q "%~dp0"..\exec.bat >nul
+set "pyenv=cscript //nologo "%~dp0..\libexec\pyenv.vbs""
+
+rem if 'pyenv' called alone, then run pyenv.vbs
+if [%1]==[] (
+  %pyenv%
+  exit /b
 )
 
-call cscript //nologo "%~dp0"..\libexec\pyenv.vbs %*
-IF EXIST "%~dp0"..\exec.bat (
-   call "%~dp0"..\exec.bat
+rem use pyenv.vbs to aid resolving absolute path of "active" version into 'bindir'
+for /f %%i in ('%pyenv% version') do call :normalizepath "%~dp0..\versions\%%i" bindir
```

Comment:
```suggestion
for /f %%i in ('%pyenv% vname') do call :normalizepath "%~dp0..\versions\%%i" bindir
```
When no version is defined, `bindir` will be `...\versions\pyenv` because `pyenv version` returns:

> No global python version has been set yet. Please set the global version by typing:
> **pyenv** global 3.7.2

Reply:
A check was added for processing `exec` command, ensuring some version is active, otherwise "No global python version has been set yet. Please set the global version." is output